go.mod

```diff
@@ -1,6 +1,6 @@
 module go.viam.com/rdk
 
-go 1.23.0
+go 1.23.8
```

Comment:
Necessary?

Reply:
not exactly sure, but https://github.com/viamrobotics/goutils/commit/edccfb87557f3dbb4bbc65134fda473b5524ad84#diff-6a350861a9b8fc96d1458c98706de34de184118fc660ed63e30bf026a97b2fa5 added it